recipes/perl-number-format/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set cpan_name = "Number-Format" %}
+{% set version = "1.76" %}
+{% set sha256 = "0e0060eb363635a885706c6a26f5fcaafeae759f7b2acae49dda70e195dd44d6" %}
+
+package:
+  name: perl-{{ cpan_name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://cpan.metacpan.org/authors/id/R/RJ/RJBS/Number-Format-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
+  script:
+    - perl Makefile.PL INSTALLDIRS=vendor NO_PERLLOCAL=1 NO_PACKLIST=1
+    - make
+    - make test
+    - make install VERBINST=1
+
+requirements:
+  build:
+    - make  # [not win]
+    - m2w64-make  # [win]
```

`m2w64` is only for `run`

```suggestion
    - m2-make  # [win]
```

---

Okay...in any case the build failed with the same error as using `make`. Do you think I should try `m2-make` again, in case the previous issue was a CI problem?